.zuul.yaml

```diff
@@ -2,7 +2,8 @@
 - job:
     name: stf-base
     # defined in: https://review.rdoproject.org/cgit/config/tree/zuul.d/_jobs-crc.yaml
-    parent: base-simple-crc
+    #parent: base-simple-crc
+    parent: base-crc
```

Comment:
This doesn't work because the custom bundle url is not parsed by the crc/start.yml playbook used by this job